stardoc/BUILD

```diff
@@ -48,38 +49,14 @@ stardoc(
     deps = [":stardoc_lib"],
 )
 
-java_binary(
-    name = "stardoc",
```

Comment:
It is possible that somebody needs //stardoc:stardoc label.
Please add an alias for it pointing to `prebuilt_stardoc_binary`